pkg/metrics/metrics.jsx

```diff
@@ -382,19 +382,25 @@ class CurrentMetrics extends React.Component {
             const merged = [];
             names.forEach((k, i) => {
                 const v = values[i];
+                let user;
                 // filter out invalid values, the empty (root) cgroup, non-services
                 if (k.endsWith('.service') && typeof v === 'number' && v != 0) {
+                    if (k.match(/^user.*user@\d+\.service.+/)) {
+                        user = true;
+                    } else {
+                        user = false;
+                    }
                     const label = k.replace(/.*\//, '').replace(/\.service$/, '');
                     // only keep cgroup basenames, and drop redundant .service suffix
-                    merged.push([label, v]);
+                    merged.push([label, v, user]);
                 }
             });
             merged.sort((a, b) => b[1] - a[1]);
             return merged.slice(0, n);
         }
 
-        function serviceRow(name, value) {
-            const name_text = <Button variant="link" isInline component="a" key={name} onClick={ e => cockpit.jump("/system/services#/" + name + ".service") }><TableText wrapModifier="truncate">{name}</TableText></Button>;
+        function serviceRow(name, value, isUser) {
+            const name_text = <Button variant="link" isInline component="a" key={name} onClick={ e => cockpit.jump("/system/services#/" + name + ".service" + (isUser ? "?owner=user" : "")) }><TableText wrapModifier="truncate">{name}</TableText></Button>;
```

Comment:
Please avoid such long lines. Break it up into the usual HTML multi-line structuring. Usually with wrapping in () to pacify the ELSlint checks. I know the original line was already quite long, but let's clean this up while touching the code anyway.